addons/core/translations/en-us.yaml

```diff
@@ -33,6 +33,7 @@ titles:
   choose-a-provider: Choose a provider
   provider: Provider
   ip-addresses: Latest IP Addresses
+  dns-names: DNS Names
```

Comment:
`dns_names` is a field, so we could move this translation into `core/translations/form/` and treat is a a label.  Similarly, `ip_addresses` is also a form field and could be moved (as a separate PR).